shared/constants/security-provider.ts

```diff
@@ -9,13 +9,16 @@ type SecurityProviderConfig = Record<
     readonly tKeyName: string;
     /** URL to securty provider website */
     readonly url: string;
+    /** URL to security provider support page */
+    readonly supportUrl: string;
   }
 >;
 
 export const SECURITY_PROVIDER_CONFIG: Readonly<SecurityProviderConfig> = {
   [SecurityProvider.Blockaid]: {
     tKeyName: 'blockaid',
     url: 'https://blockaid.io/',
+    supportUrl: 'https://support.metamask.io/hc/en-us',
```

Comment:
[nit] would be nice to keep alphabetical in case things grow or change. same comment for SecurityProviderConfig above